utils/comment.go

```diff
@@ -44,10 +44,12 @@ func HandlePullRequestCommentsAfterScan(issues *IssuesCollection, repo *Reposito
 	}
 
 	// Add summary (SCA, license) scan comment
-	for _, comment := range generatePullRequestSummaryComment(issues, repo.OutputWriter) {
-		if err = client.AddPullRequestComment(context.Background(), repo.RepoOwner, repo.RepoName, comment, pullRequestID); err != nil {
-			err = errors.New("couldn't add pull request comment: " + err.Error())
-			return
+	if repo.AddPrCommentOnSuccess {
```

Comment:
looks like it will not add comments at all - If there are issues - I do want to add them - the purpose here is to skip on adding comments only if no issues were found.

In that case - maybe we need to change the env var name - to something like "skip_no_issues_comment" (default=False)

Reply:
nice, fixed